<?xml version="1.0"?>
<launch>
  <!-- IP and port on Vicon Windows PC -->
<!--  <arg name="datastream_hostport" default="10.250.231.90"/>-->
    <arg name="datastream_hostport" default="192.168.1.52"/>
  <!-- Fixed vicon world frame -->
    <arg name="vicon_world_frame" default="world"/>
    <arg name="subject_name" value="p450"/>
    <arg name="segment_name" value="p450"/>

	<node pkg="uav_localization" type="vicon_bridge" name="vicon" output="screen" >
		<param name="stream_mode" value="ServerPush"/>
    <param name="datastream_hostport" value="$(arg datastream_hostport)"/>
		<param name="tf_ref_frame_id" value="$(arg vicon_world_frame)"/>
	</node>

    <node pkg="uav_localization" type="mavros_bridge" name="mavros_bridge" output="screen" >
        <param name="subject_name" value="$(arg subject_name)"/>
        <param name="segment_name" value="$(arg segment_name)"/>
		<param name="child_frame_id" value="base_link"/>
        <param name="frame_id" value="odom"/>
	</node>
</launch>
